installers-conda/build_conda_pkgs.py

```diff
@@ -334,23 +323,20 @@ class PylspCondaPkg(BuildCondaPkg):
     source = os.environ.get('PYTHON_LSP_SERVER_SOURCE')
     feedstock = "https://github.com/conda-forge/python-lsp-server-feedstock"
     feedstock_branch = "main"
-    shallow_ver = "v1.4.1"
 
 
 class QtconsoleCondaPkg(BuildCondaPkg):
     name = "qtconsole"
     source = os.environ.get('QTCONSOLE_SOURCE')
     feedstock = "https://github.com/conda-forge/qtconsole-feedstock"
     feedstock_branch = "main"
-    shallow_ver = "5.3.1"
 
 
 class SpyderKernelsCondaPkg(BuildCondaPkg):
     name = "spyder-kernels"
     source = os.environ.get('SPYDER_KERNELS_SOURCE')
     feedstock = "https://github.com/conda-forge/spyder-kernels-feedstock"
-    feedstock_branch = "rc"
-    shallow_ver = "v2.3.1"
+    feedstock_branch = "main"
```

Comment:
As you know, we also have a `dev` branch in Spyder-kernels and it could diverge from the `main` one for 6.1 (if not, that could happen for a later version too).

So, perhaps you should also select here the feedstock branch according to the Spyder one.

Reply:
Great suggestion. I'll use the same logic: spyder master -> spyder-kernels-feedstock dev; spyder 6.x -> spyder-kernels-feedstock main